content/en/docs/armory-admin/armory-agent/agent-monitoring.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Monitoring
+weight: 5
+---
+
+## Prometheus
+
+If `prometheus.enabled` is true, the Agent exposes metrics on port 8008 (`prometheus.port`) on path `/metrics` or `/prometheus_metrics` like other Spinnaker services. Both paths serve the same data.
```

Comment:
`prometheus.enabled` where?

Reply:
It is explained in the agent options. Perhaps link there?